app/templates/components/form/responses-table.hbs

```diff
@@ -5,7 +5,7 @@
     {{/if}}
     <th> Naam </th>
     {{#each form.sortedQuestions as |question|}}
-      <th> {{question.question}} </th>
+      <th> {{question.question}} - {{question.position}} </th>
```

Comment:
This starts with 0

Maybe a nicer way of doing this is showing it in the Inschrijven form as "{{pos}}. {{question}}" and then have that format here as well?

Reply:
No it shouldn't show the positon, that was for debug, will remove it